Misc/ACKS

```diff
@@ -1837,3 +1837,4 @@ Jelle Zijlstra
 Gennadiy Zlobin
 Doug Zongker
 Peter Åstrand
+Juliette Monsel
```

Comment:
And the last. Names in this list are in rough alphabetical order by last names. Move your name to the correct place.

Reply:
Sorry for all the mistakes and thanks for taking the time to point them out, I'll try to do everything right from the start next time.